maps/outreach/outreach_machinery.dm

```diff
@@ -0,0 +1,219 @@
+		/obj/item/stock_parts/radio/receiver/buildable
+	)
+
+////////////////////////////////////////////////////////////////////////
+// Blast Doors
+////////////////////////////////////////////////////////////////////////
+/obj/machinery/door/blast/regular/wired
+	stock_part_presets = list(
+		/decl/stock_part_preset/terminal_setup,
+		/decl/stock_part_preset/radio/receiver/blast_door,
+	)
+	uncreated_component_parts = list(
+		/obj/item/stock_parts/power/terminal,
+		/obj/item/stock_parts/radio/receiver/buildable,
+	)
+
+/obj/machinery/door/blast/regular/wired/open
+	icon_state = "pdoor0"
+	begins_closed = FALSE
+
+
+////////////////////////////////////////////////////////////////////////
+// Suit Cyclers
+////////////////////////////////////////////////////////////////////////
+/obj/machinery/suit_cycler/emergency/prepared
+	name                    = "work suit cycler unit"
+	buildable               = FALSE
+	initial_access          = list()
+	helmet                  = /obj/item/clothing/head/helmet/space/emergency
+	suit                    = /obj/item/clothing/suit/space/emergency
+	boots                   = /obj/item/clothing/shoes/workboots
+	available_bodytypes     = list(BODYTYPE_HUMANOID, BODYTYPE_MONKEY)
+	available_modifications = list(
+		/decl/item_modifier/space_suit/engineering,
+		/decl/item_modifier/space_suit/mining,
+		/decl/item_modifier/space_suit/medical,
+		/decl/item_modifier/space_suit/security,
+		/decl/item_modifier/space_suit/atmos,
+		/decl/item_modifier/space_suit/science,
+		/decl/item_modifier/space_suit/pilot,
+		/decl/item_modifier/space_suit/salvage,
+	)
+
+////////////////////////////////////////////////////////////////////////
+// SMES
+////////////////////////////////////////////////////////////////////////
+/obj/machinery/power/smes/buildable/preset/outreach
+	_fully_charged = TRUE
+	_input_maxed   = TRUE
+	_input_on      = TRUE
+	_output_maxed  = FALSE
+	_output_on     = TRUE
+	output_level   = 150000
+	maximum_component_parts = list(
+		/obj/item/stock_parts = 20,
+	)
+	uncreated_component_parts = list(
+		/obj/item/stock_parts/smes_coil/super_capacity = 5,
+		/obj/item/stock_parts/smes_coil/super_io       = 1,
+	)
+
+/obj/machinery/power/smes/buildable/preset/outreach/substation
+	_fully_charged = TRUE
+	_input_maxed   = TRUE
+	_input_on      = TRUE
+	_output_maxed  = FALSE
+	_output_on     = TRUE
+	output_level   = 100000
+	uncreated_component_parts = list(
+		/obj/item/stock_parts/smes_coil/super_capacity = 2,
+		/obj/item/stock_parts/smes_coil/super_io       = 1,
+	)
+
+////////////////////////////////////////////////////////////////////////
+// Docking Beacon
+////////////////////////////////////////////////////////////////////////
+/obj/machinery/docking_beacon/mapped
+	anchored = TRUE
+
+////////////////////////////////////////////////////////////////////////
+// Defense Turret
+////////////////////////////////////////////////////////////////////////
+/obj/machinery/porta_turret/stationary/stun
```

Comment:
Making a note that we'll need to replace these with the new turrets once they're in